### ISSUE TITLE
File chooser | directory navigation issue

### ISSUE THREAD

**Issue by RockyRx** [kivy/kivy#6034]
Not sure if this is a bug or not but directory navigation works with a single click. This is happening on Windows 10 when using the touchpad. Ideally, directory selection/navigation should happen when double-clicked.

### Versions
* Python: 3.7.0
* OS: Windows 10
* Kivy: 1.10.1
* Kivy installation method: `pip install kivy`

**Comment by matham** [comment]
Yeah, this is something that I always found annoying. However, it's unlikely to be fixed in the current implementation because filechooser and the underlying treeview needs a complete rewrite IMO and I'm not sure it can be fixed in the current implementation.

**Comment by RockyRx** [comment]
@matham Is there any alternative solution for this matter?
